flashinfer/mxfp8_attention_sm120.py

```
@@ -0,0 +1,439 @@
+    r"""Ragged varlen FP8 prefill attention on SM120/SM121 (one-shot plan+run).
+
+    Convenience wrapper around :class:`MXFP8AttentionSM120Wrapper` for single-shot
+    use; serving integrations should prefer the wrapper so the per-batch host work
+    (padding layout + LPT work lists) is amortized across layers. See
+    :meth:`MXFP8AttentionSM120Wrapper.run` for the full contract.
+
+    .. note::
+        This function moves ``qo_indptr``/``kv_indptr`` to the host (one sync)
+        on every call.
+    """
+    total_q, num_qo_heads, _ = q.shape
```

Ruff (0.15.21)

[warning] Unpacked variable `total_q` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)